.github/workflows/build.yml

```diff
@@ -12,14 +12,12 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: ${{ matrix.go }}
+
       - name: Check out source
         uses: actions/checkout@v1
       - name: Install Linters
         run: "curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0"
-      - name: Build
-        env:
-          GO111MODULE: "on"
-        run: go build -race ./...
```

Comment:
I thought I had left a self-review comment here, but I didn't click submit.

This test appeared to be doing nothing. Is recursive building a thing? Works better in *run_tests.sh* anyways, since I typically run that before pushing.

Reply:
`./...` will do all packages under the path in the same module, just not others like the cmd modules.  I think it's good to do somewhere if it's here or in run_tests.sh

Reply:
So chatting on Matrix, I think we decided that since the tests force compilation of all package code anyway then the `go build ./...` in the repo root doesn't add anything.  It's only useful to go build the `main` packages.